### PR TITLE
Wow, that's some messy code!

### PR DESCRIPTION
**Pull request by inz-**

### CODE REVIEW

File: Src/BootCamp.Chapter1/ArrayOperations.cs [hunk 1]
```diff
@@ -7,19 +12,30 @@ public static class ArrayOperations
...
         /// If array empty or null- don't do anything.
         /// </summary>
         /// <param name="array">Input array in a random order.</param>
-        public static void Reverse(int[] array)
+        public static int[] Reverse(int[] array)
         {
-            // ToDo: implement.
+            if (array == null)
+                return array;
+            Array.Reverse(array);
```
Comment (Almantask) [inline Src/BootCamp.Chapter1/ArrayOperations.cs]: Don't cheat! 😄 Please implement this by yourself. No C#-unique language features are yet allowed.

File: Src/BootCamp.Chapter1/ArrayOperations.cs [hunk 2]
```diff
@@ -7,19 +12,30 @@ public static class ArrayOperations
...
+        public static int[] Sort(int[] array)
         {
-            // ToDo: implement.
+            if (array == null)
+            {
+                return array;
+            }
+            else
+            {
+                Array.Sort(array);
```
Comment (Almantask) [inline Src/BootCamp.Chapter1/ArrayOperations.cs]: Implement this yourself 😄

**Review by Almantask — CHANGES_REQUESTED** [review]
Please redo the homework by understanding how to implement each function yourself, and not using the language features yet.

File: Src/BootCamp.Chapter1/ArrayOperations.cs [hunk 3]
```diff
@@ -45,14 +65,17 @@ public static int[] Reverse(int[] array)
         /// <returns>A new array with the last element removed. If an array is empty or null, returns input array.</returns>
         public static int[] RemoveLast(int[] array)
```
Comment (Almantask) [inline Src/BootCamp.Chapter1/ArrayOperations.cs:66]: RemoveLast is actually RemoveAt(last)

File: Src/BootCamp.Chapter1/ArrayOperations.cs [hunk 4]
```diff
@@ -61,14 +84,17 @@ public static int[] RemoveLast(int[] array)
         /// <returns>A new array with the first element removed. If an array is empty or null, returns input array.</returns>
         public static int[] RemoveFirst(int[] array)
```
Comment (Almantask) [inline Src/BootCamp.Chapter1/ArrayOperations.cs:79]: RemoveAt(0)

**Review by Almantask — CHANGES_REQUESTED** [review]
A lot of methods can be reused when you removeAt or InsertAt a given index.

File: Src/BootCamp.Chapter1/ArrayOperations.cs (hunk 3 above)
Comment (inz-) [reply Src/BootCamp.Chapter1/ArrayOperations.cs:66]: thanks for pointing out the obvious :)

**Comment by inz-** [comment]
This should have been obvious, but thanks ;)
Has been changed

**Review by Almantask — APPROVED** [review]
Looks good

**Pull request closed without merging**